packages/flutter/lib/src/widgets/framework.dart

```diff
@@ -258,23 +258,6 @@ class GlobalObjectKey<T extends State<StatefulWidget>> extends GlobalKey<T> {
   }
 }
 
-/// This class is a work-around for the "is" operator not accepting a variable value as its right operand.
```

Comment:
This is not ready for removal. I just tried to get this out in https://github.com/flutter/flutter/pull/73751 but it was reverted (https://github.com/flutter/flutter/pull/75734) due to internal test failures. I'm working on migrating those after which point this can be removed.